com.unity.multiplayer.mlapi/Editor/CodeGen/NetworkBehaviourILPP.cs

```diff
@@ -492,8 +543,9 @@ private void InjectWriteAndCallBlocks(MethodDefinition methodDefinition, CustomA
                 var returnInstr = processor.Create(OpCodes.Ret);
                 var lastInstr = processor.Create(OpCodes.Nop);
 
-                // networkManager = NetworkManager.Singleton;
-                instructions.Add(processor.Create(OpCodes.Call, NetworkManager_getSingleton_MethodRef));
+                // networkManager = this.NetworkManager;
+                instructions.Add(processor.Create(OpCodes.Ldarg_0));
+                instructions.Add(processor.Create(OpCodes.Call, NetworkBehaviour_getNetworkManager_MethodRef));
```

Comment:
Get `NetworkManager` from `NetworkBehaviour.NetworkManager` property instead of `NetworkManager.Singleton`